_posts/en/posts/2024-07-03-disclose_already_asked_for.md

```diff
@@ -7,7 +7,7 @@ type: advisory
 layout: post
 
 ## If this is a new post, reset this counter to 1.
-version: 1
+version: 2
```

Comment:
Am I doing this right? Should the version be incremented?

Reply:
Will double-check, but I think this might be related to translations. In either case, website content has updated ok. i.e: https://bitcoincore.org/en/2024/07/03/disclose-bip70-crash/.